test/src/401-ducc-convert_singularity/main

```diff
@@ -0,0 +1,16 @@
+cvmfs_test_name="DUCC singularity image conversion"
+cvmfs_test_autofs_on_startup=true
+cvmfs_test_suites="ducc"
+
+DOCKER_TEST_IMAGE="https://registry.hub.docker.com/library/centos:centos7"
+
+
+cvmfs_run_test() {
+  echo "*** creating empty repo ***"
+  create_empty_repo $CVMFS_TEST_REPO $CVMFS_TEST_USER || return 1
+
+  echo "*** converting image to singularity ***"
+  cvmfs_ducc convert-singularity-image --repo $TEST_REPO $DOCKER_TEST_IMAGE || return 2
+
```

Comment:
There should be some test code here to check that the image contents actually appeared in the repository